tinker/tinker_controller.py

```diff
@@ -190,6 +191,11 @@ def admin_permissions(flask_view_class):
         if 'Administrators' not in session['groups']:
             abort(403)
 
+    # TODO add in the user roles menu so help desk can see it
```

Comment:
is this todo finished?

Reply:
yeah, I forgot to change that group to the right IAM group so I fixed that in the most recent commit and removed the TODO